devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -206,6 +208,8 @@ did-auth-proxy-helm:
             set $initial_content_type $sent_http_content_type;
             add_header Content-Type ""  always;
             add_header Content-Type $cors_content_type  always;
+
```

Comment:
redeclaring header, because it is not inherited from enclosing block https://www.nginx.com/blog/http-strict-transport-security-hsts-and-nginx/